### DISCUSSION TITLE
Disconnect Client from Server

### DISCUSSION THREAD

**Discussion by sefaa-dev** [dotnet/MQTTnet#1838] [General]
hi.
Can we disconnect from the server for a client?
for example if user session expierd in server i want to drop connection

**Comment by hgraghu** [comment]
Yes you can disconnect a client from server by passing clientname and reason

**Comment by sefaa-dev** [comment]
which function do this ?

**Comment by hgraghu** [comment]
mqttServer.DisconnectClientAsync(e.ClientId, dusconnectionoptions)

Look for mqttclientdisconnectionoptions

**Comment by sefaa-dev** [comment]
Thankyou
